pydatarecognition/fsclient.py

```diff
@@ -234,7 +239,7 @@ def all_documents(self, collname, copy=True):
             return deepcopy(self.chained_db.get(collname, {})).values()
         return self.chained_db.get(collname, {}).values()
 
-    def insert_one(self, filename, dbname, collname, doc):
+    def insert_one(self, filename, doc):
```

Comment:
do we know why this takes `filename` and doesn't take dbname and collname?  Doesn't htis break the api, because the `insert_one` will have a different signature for the different clients (mongo vs fs).

I may be wrong here, so let me know if so, but I htink the idea is that `insert_one` has the same signature regardless of the client, but it has different behavior depending on the backend.  In general, we want to insert something into a particular collection in a particular database.  If that collection is in a mongo db the client will need a URL and login credentials and so on of the db, if it is in a file on a filesystem it will need to know the file-name and where in the directory structure it is located, but this is handled by the `rc.databases` at runtime.